app/javascript/packages/document-capture/context/acuant.jsx

```diff
@@ -63,18 +65,21 @@ function AcuantContextProvider({
   endpoint = null,
   children,
 }) {
-  const [isReady, setIsReady] = useState(false);
+  const { isMobile } = useContext(DeviceContext);
+  const [isReady, setIsReady] = useState(!isMobile);
+  const [isAcuantLoaded, setIsAcuantLoaded] = useState(false);
   const [isError, setIsError] = useState(false);
-  const [isCameraSupported, setIsCameraSupported] = useState(/** @type {?boolean} */ (null));
-  const value = useMemo(() => ({ isReady, isError, isCameraSupported, endpoint, credentials }), [
-    isReady,
-    isError,
-    isCameraSupported,
-    endpoint,
-    credentials,
-  ]);
+  const [isCameraSupported, setIsCameraSupported] = useState(isMobile ? null : false);
```

Comment:
what's the difference between `useState(null)` and `useState(false)` here?

Reply:
> what's the difference between `useState(null)` and `useState(false)` here?

I'll plan to add some inline code comments here. Essentially:

- If the user is on a mobile device, we don't know whether the camera is supported until after Acuant SDK loads and  we call `AcuantCamera.isCameraSupported`. Until then, `null` stands for a value of " :shrug: " (unknown).
- On other device types, since we're bypassing Acuant SDK load, we set it immediately to `false`, since in all situations where Acuant SDK camera availability would be relevant, we want for it to be treated as if it's not. Though, truth be told, it shouldn't matter too much if it were `true` or `false` or `null`, since it shouldn't ever be referenced if `isReady && !isAcuantLoaded`.

Reply:
> > what's the difference between `useState(null)` and `useState(false)` here?
> 
> I'll plan to add some inline code comments here.

Added in d2dcd7809.

Also, small correction to my comment above: It does matter what value we use, because we rely on `isCameraSupported` as part of [a condition](https://github.com/18F/identity-idp/blob/1acb12a6c167d9e6157b8807c13c94a1887fd0d0/app/javascript/packages/document-capture/components/acuant-capture.jsx#L180) which considers [whether to render `<AcuantCaptureCanvas />`](https://github.com/18F/identity-idp/blob/1acb12a6c167d9e6157b8807c13c94a1887fd0d0/app/javascript/packages/document-capture/components/acuant-capture.jsx#L336-L346), since that component will [assume the presence of Acuant globals](https://github.com/18F/identity-idp/blob/1acb12a6c167d9e6157b8807c13c94a1887fd0d0/app/javascript/packages/document-capture/components/acuant-capture-canvas.jsx#L118). In earlier iterations I'd revised that component to consider `isAcuantLoaded` instead of `isReady`, though it wasn't strictly necessary to do so. Other tests to catch this incidentally if it was misconfigured, but I added an explicit test case in cf434b914 + ec182c338.